schemas/content/1.0/schema.json

```diff
@@ -1017,6 +1017,9 @@
         },
         "board" : {
             "type": "string"
+        },
+        "category": {
+            "type": "string"
```

Comment:
Remove the content or collection integration changes in this PR. Let us do that separately once we get full clarity of the requirement.